pkg/eks/update.go

```diff
@@ -51,6 +53,9 @@ type UpdateResourceTagsOpts struct {
 func UpdateResourceTags(ctx context.Context, opts *UpdateResourceTagsOpts) (bool, error) {
 	updated := false
 	if updateTags := utils.GetKeyValuesToUpdate(opts.Tags, opts.UpstreamTags); updateTags != nil {
+		logrus.Infof("updating resource tags to %s for cluster [%s]", opts.Tags, opts.ClusterName)
+		logrus.Debugf("config: %s, upstream: %s", opts.Tags, opts.UpstreamTags)
```

Comment:
I have doubts if this will work with `%s`

Reply:
%s works here since `opts.Tags` is a map[string]string, but it is better to change this to %v in any case, thank you for catching this!